Storage.md

```diff
@@ -0,0 +1,291 @@
+
+HTML elements stored within ItemsHoldr will have their values on the page updated to their changed value.
+
+```typescript
+ItemsHolder.addItem("color", {
+    hasElement: true,
+    element: {}
+});
+ItemsHolder.setItem("color", "red");
+```
+
+To signal if a container should be made to hold HTML elements, assign a value to `doMakeContainer`.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({ doMakeContainer: true });
+```
+
+With this, any HTML items passed in at the time of construction are held in ItemsHolder's `container` object.
+Each of the item's elements are added to the container as its children.
+It can be used to display a number of elements grouped together.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({
+    doMakeContainer: true,
+    values: {
+        color: {
+            valueDefault: "red",
+            hasElement: true,
+            element: {}
+        }
+    }
+});
+
+// color is in the container, but speed is not
+ItemsHolder.addItem("speed", {
+    valueDefault: 10,
+    hasElement: true,
+    element: {}
+});
+```
+
+### Display Changes
+
+If you want certain values to be represented differently on the page, make use of `displayChanges`.
+These changes are hardcoded values that replace specified values when element items are updated.
+
+```typescript
+let ItemsHolder = new ItemsHoldr({
+    displayChanges: {
+        Infinity: "INF";
+    }
+});
+
+ItemsHolder.addItem("limit", {
+    value: "4000",
+    hasElement: true,
+    element: {}
+});
+ItemsHolder.setItem("limit", "Infinity");
+```
+
+`hasDisplayChange` checks to see if a value has a recorded change.
+
+```typescript
+ItemsHolder.hasDisplayChange("Infinity");
+```
+
+...and `getdisplayChange` returns the entry to replace the value with.
+
+```typescript
+let newValue: string = ItemsHolder.getDisplayChange("Infinity");
+```
+
+### Container Visibility
+
+`hideContainer` hides the container from view.
+
+```typescript
+ItemsHolder.hideContainer();
+```
+
+...while `displayContainer` makes the container visible.
+
+```typescript
+ItemsHolder.displayContainer();
+```
+
+More can be read about ItemsHoldr on its [Readme](https://github.com/FullScreenShenanigans/ItemsHoldr/blob/master/README.md).
+
+# StateHoldr
+
+StateHoldr is a module for tracking changes of items in ItemsHolder in objects called collections.
+
+Like ItemsHoldr, StateHoldr has its own prefix property.
+
+```typescript
+let StateHolder = new StateHoldr({
+    ItemsHolder: new ItemsHoldr(),
+    prefix: "StateHolder"
+});
+```
+
+## Changes
+
+Changes are stored as key-value pairs under an item.
+To add a change to the current collection, use `addChange`.
+`addChange` takes in the key of the item, an attribute of the item, and the value being stored.
+
+```typescript
+StateHolder.addChange("car", "color", "red");
+```
+
+If the name of the collection is known, whether or not its the current collection, a change can be added with `addCollectionChange`.
```

Comment:
It would be good to mention before here that there is such a concept as a "current" collection.